src/components/Registration/NaicsResult.vue

```diff
@@ -1,11 +1,14 @@
 <template>
-  <v-row no-gutters class="naics-result" v-if="!!result" @click="emitClick(result)">
```

Comment:
No need to emit `result`: parent already know which result this is (to select it).